ppb/events.py

```diff
@@ -98,6 +109,23 @@ class ButtonReleased:
     scene: Scene = None
 
 
+@dataclass
+class ChangeScene:
+    """
+    Fired to start a new scene.
+
+    new_scene can be an instance or a class. If a class, must include kwargs.
+    If an instance kwargs must be falsey, default is None.
+
+    Examples:
+        * `signal(new_scene=ChangeScene(MyScene(player=player))`
+        * `signal(new_scene=ChanngeScene, kwargs={"player": player}`
```

Comment:
I kinda wish this form had a calling form, like `SomethingSomething(ChangeScene, player=player)`.

Also, typo

Reply:
We could make it _not_ a dataclass? Then I can just add kwargs to that signature.

It makes it feel weird because it'll only be a handful of events that do that.